cumulusci/tasks/metadata_etl/layouts.py

```diff
@@ -68,6 +69,262 @@ def _create_new_related_list(self, metadata, api_name, related_list):
+        # Do nothing if there are no pages
+        if len(self._adding_pages) == 0:
+            return
+
+        page_props = {page["api_name"]: page for page in self._adding_pages}
+
+        layout_item_dict = self._add_items(self._adding_pages, api_name)
+
+        for page_item_key in layout_item_dict.keys():
+            page = page_props.get(page_item_key)
+            page_layout_item = layout_item_dict.get(page_item_key)
+            adding_page_name = page.get("api_name")
+
+            self.logger.info(f"Adding {adding_page_name} to {api_name}")
+
+            page_layout_item.append("page", adding_page_name)
+            page_layout_item.append("height", str(page.get("height", 200)))
+            page_layout_item.append(
+                "showLabel",
+                str(process_bool_arg(page.get("show_label") or False)).lower(),
+            )
+            page_layout_item.append(
+                "showScrollbars",
+                str(process_bool_arg(page.get("show_scrollbars") or False)).lower(),
+            )
+            page_layout_item.append("width", page.get("width", "100%"))
+
+    def _add_items(self, item_list, api_name):
+        """Iterate over a list of new layout items that will return a dict{api_name:MetadataElement}"""
+        ## If no items are passed in, do nothing
+        if len(item_list) == 0:
+            return
+
+        item_dict = {}
+
+        for item in item_list:
+            item_name = self._inject_namespace(item.get("api_name"))
+            if item_name in self._existing_names:
+                self.logger.warning(
+                    f"Skipped {item_name} because {item_name} is already present in {api_name}"
+                )
+                continue
+
+            item_dict[item_name] = self._position_item(item)
+
+        return item_dict
+
+    def _position_item(self, new_item):
+        """Returns new MetadataElement being added to the layout based on the positioning properties"""
+        new_item_name = new_item.get("api_name")
+        # Default to bottom of first section (index zero) of first column
+        default = {
+            "relative": "top",
+            "section": 0,
+            "column": "last",
+        }
+
+        # Position is optional
+        position = new_item.get("position")
+
+        if position is None:
+            self.logger.warning(
+                f"Position details are missing for: {new_item_name}. Default position is being applied."
+            )
+            position = [default]
+
+        # append the default position to the list as a fallback
+        position.append(default)
+
+        for pos in position:
+            new_layout_item = self._process_position(pos, default)
+            if new_layout_item is not None:
+                return new_layout_item
+
+        # If layout section doesn't exist, exit gracefully with appropriate message
+        if new_layout_item is None:
+            raise CumulusCIException(
```

Comment:
Is this possible, given our default above? Would that mean a page layout that has no sections (and is that even possible?)

Reply:
I questioned this myself, and settled on keeping it "just in case" but it certainly makes sense to remove it.